Seagate/cortx-utils
# CORTX-29993 - Update cryptography library from 2.8 to 3.3.2

# Problem Statement
Current cryptography version is 2.8. This version has High/medium level vulnerabilities
This version is planned to be upgraded to 3.3.2

Thought process of upgrading to version 3.3.2 is as follows

Version upgrade for cryptography library from version 2.8 to version 3.3.2 is recommended by Whitesource vulnerability scanning tool.
There are several cryptography versions available which are higher than 3.3.2. However further library package upgrade will be planned in subsequent iteration,

# Design
-  For Bug describe the fix here. 
-  For Feature, Post the link to the solution page on the confluence CORTX Foundation Library 

Reference to cryptography version to 3.3.2 is upgraded at following file
– py-utils/python_requirements.txt

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing
- Are test cases updated along with code changes due to Enhancements/Bugs [Y/N]: Y
- Confirm that new test cases are added to regression and sanity plan files and relevant feature plan files [Y/N]: Y
- Confirm that Test Cases are added for new features added [Y/N]:  Y
- Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]:  Y
- Confirm Testing was performed with installed RPM/K8s deployment [Y/N]:   Y

Detailed manual and regression testing is complete and test results are uploaded to this story
https://jts.seagate.com/browse/CORTX-29817

# Review Checklist 
####  Before posting the PR please ensure:
- [X] If yes for above point, Is a notification sent to all other cortx components [Y/N] N
- [X] New package/s added to setup.py? No
- [X] Jira is updated : Y

# KVstore/Confstore feature (changes/fixes) checklist
#### Confirm changes are made for:

Changes are not applicable to KVstore or Confstore

# Documentation

Not applicable
